clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -2221,3 +2221,11 @@
       Out: ""
       MatchMode: Full
       RuleId: "remove_empty_library_arg"
+
+- Rule: rule_set_source_files_properties_append
```

Comment:
Rule name can be:
rule_set_source_files_properties

Don't see the need for append